slides/reveal-js/technical_introduction_20240912/index.html

```diff
@@ -0,0 +1,343 @@
+                                Implementierungsaufwände der dezentralen Kommunikation (ATF) entfallen für zukünftige
+                                Use Cases.</li>
+                            <li><strong>Einfachere Erweiterung um weitere Use Cases:</strong> Die Spezifikation und
+                                Umsetzung neuer Use Cases werden vereinfacht, da die Notwendigkeit eines zentralen
+                                Dienstes entfällt.</li>
+                        </ul>
+                    </div>
+                </section>
+
+            </section>
+
+            <section>
+                <h3>Welches Problem löst das ATF?</h3>
+                <div style="font-size:0.75em; margin-top:0px; flex: 1; margin-left: 20px;">
+                    <p><strong>Herausforderung der Validierung in Peer-to-Peer Netzwerken</strong></p>
+                    <p>Im Gegensatz zu zentralisierten Systemen, wo ein zentraler Server die Validierung eingehender
+                        Informationen übernimmt, fehlt in Peer-to-Peer Netzwerken ein solcher zentraler Kontrollpunkt.
+                        Dies führt zu spezifischen Herausforderungen:</p>
+                    <ul>
+                        <li><strong>Dezentrale Validierung:</strong> Jeder Teilnehmer muss die Informationen
+                            selbstständig validieren, klare Kommunikationsprotokolle zu aufgetretenen Fehlern und deren
+                            Lösung
+                            müssen definiert sein.</li>
+                        </li>
+                        <li><strong>Standardisierter Austausch:</strong> Das ATF schafft einen Rahmen, innerhalb dessen
+                            Sender und Empfänger Informationen über die Validität und den Verarbeitungsstatus von
+                            Nachrichten austauschen können.</li>
+                        <li><strong>Lösung von Konflikten:</strong> Das Framework ermöglicht es, auf effiziente Weise
+                            festzustellen, ob Nachrichten korrekt verarbeitet wurden und bei Problemen den Ursprung
+                            schnell
+                            zu identifizieren.</li>
+                    </ul>
+                </div>
+            </section>
+
+            <section>
+
+                <section>
+                    <h3>Wie funktioniert das ATF?</h3>
+                    <div style="font-size:0.75em; margin-top:0px; flex: 1; margin-left: 20px;">
+                        <p><strong>Kernfunktionen und Prozesse:</strong></p>
+                        <p>Das ATF nutzt standardisierte Prozesse für die Systemkommunikation, die speziell auf einzelne
+                            Anwendungsfälle zugeschnitten sind:</p>
+                        <ul>
+                            <li><strong>Anwendungsfall-Repräsentation:</strong> Jeder <a
+                                    href="https://simplifier.net/app-transport-framework/app-tags-cs"
```

Comment:
Woher kommt der Inhalt dieser Liste?
Hier wird m.E. der Eindruck vermittelt , dass bereits feststeht, dass ATF hier angewendet wird.
Da aber bspw. auch die Kommunikation mit Krankenkassen hier aufgeführt ist und wir diese noch nicht in die Diskussion eingebunden haben, sehe ich das nicht als Selbstläufer an.

Reply:
Ggf. müssen wir hier ein Auszugs-ValueSet anlegen, was nur die mit ATF umgesetzten Anwendungsfälle listet. Idealerweise würde dann Stück für Stück ein Umzug stattfinden.
@SvenSommer soll ich das noch bis zur Präsentation anlegen?

Reply:
Liste ist mit Julia abgestimmt.